aesara/compile/compilelock.py

```diff
@@ -45,15 +45,15 @@ def force_unlock(lock_dir: os.PathLike):
 
 
 @contextmanager
-def lock_ctx(lock_dir: os.PathLike = None, *, timeout: Optional[float] = None):
+def lock_ctx(lock_dir: Optional[os.PathLike] = None, *, timeout: Optional[float] = -1):
```

Comment:
```suggestion
def lock_ctx(lock_dir: Optional[os.PathLike] = None, *, timeout: Optional[float] = None):
```
It was intentionally `None`, because this way it defaults to the config setting.

Reply:
+1